### PR TITLE
HAWQ-40. Refact hawq standby init. Changes to standby int and start.

### PR DESCRIPTION
**Pull request by radarwave**
Refact hawq standby init. Change standby start sequence and do init directly on standby host(previously on master node).

### CODE REVIEW

File: tools/bin/hawq_ctl [hunk 1]
```diff
@@ -149,7 +153,7 @@ class HawqInit:
               (source_hawq_env, self.vseg_num_per_node)
         result = local_ssh(cmd, logger)
         if result != 0:
-            logger.warn("Set hawq_resourcemanager_query_vsegment_number_per_segment_limit failed.")
+            logger.warn("Set hawq_resourcemanager_query_vsegment_number_per_segment_limit failed")
         return result
```
Comment (yaoj2) [inline tools/bin/hawq_ctl]: hawq_resourcemanager_query_vsegment_number_per_segment_limit guc name change, you should use new guc name

File: tools/bin/hawq_ctl [hunk 2]
```diff
@@ -284,18 +308,10 @@ class HawqInit:
             logger.info("Segments init successfully on nodes '%s'" % self.host_list)
 
         if self.standby_host_name not in ('', 'None', 'none', 'NONE'):
```
Comment (yaoj2) [inline tools/bin/hawq_ctl]: We should lower hostname first and then compare with none

**Comment by yaoj2** [comment]
Looks good to me.

**Comment by huor** [comment]
The fix looks good. +1

**Comment by denalex** [comment]
!! Please hold off on committing this change until after the Beta. Ambari HAWQ plugin will not be able to accommodate these changes for the Beta release. !!

**Comment by radarwave** [comment]
@denalex  Ok, this commit is in, I will revert it.

**Comment by radarwave** [comment]
@denalex Please let me know if this change can be pushed in now since beta is released. Thanks.

**Comment by vVineet** [comment]
Sure, please go ahead.

**Pull request closed without merging**